src/pages/help-centre.js

```diff
@@ -1,13 +1,688 @@
+                            onClick={onClick}
+                            link_style={{ size: '2rem' }}
+                        />
+                    </ListWrapper>
+                )}
+            </LeftRightContainer>
+        </>
+    )
+}
+Article.propTypes = {
+    all_articles: PropTypes.array,
+    article: PropTypes.object,
+    onClick: PropTypes.func,
+    toggleSearch: PropTypes.func,
+}
+
+const SearchSuccess = ({ suggested_topics, onClick, max_length }) => (
+    <>
+        <Header as="h3" color="white">
+            {localize('Topic Suggestions')}
+        </Header>
+        <LinkList
+            list={suggested_topics.slice(0, max_length)}
+            onClick={onClick}
+            link_style={{ color: 'white', size: '2rem' }}
+        />
+    </>
+)
+SearchSuccess.propTypes = {
+    max_length: PropTypes.number,
+    onClick: PropTypes.func,
+    suggested_topics: PropTypes.array,
+}
+
+const Ul = styled.ul`
+    list-style: unset;
+    color: var(--color-white);
+    font-size: var(--text-size-s);
+    margin-top: 0.8rem;
+    margin-left: 2rem;
+
+    > *:not(:last-child) {
+        padding-bottom: 0.8rem;
+    }
+`
+
+const Ol = styled(Ul).attrs({
+    as: 'ol',
+})`
+    list-style-type: decimal;
+`
+
+const Li = styled(Text).attrs({
+    as: 'li',
+})``
+
+const ErrorHeader = styled(Header)`
+    font-size: 2rem;
+    padding-bottom: 0.8rem;
+`
+
+const SearchError = ({ search }) => (
+    <>
+        <ErrorHeader as="h5" color="white">
+            {localize(
+                `Sorry, we couldn’t find any results matching “${search}”.`,
+                { search },
+            )}
+        </ErrorHeader>
+        <Text color="green">{localize('Search tips:')}</Text>
+        <Ul>
+            <Li color="white">
+                {localize('Check your spelling and try again')}
+            </Li>
+            <Li color="white">{localize('Try another keyword')}</Li>
+            <Li color="white">
+                {localize(
+                    'Keep your search term short as our search capabilities works best with short search terms',
+                )}
+            </Li>
+        </Ul>
+    </>
+)
+SearchError.propTypes = {
+    search: PropTypes.string,
+}
+
+const ArticleList = ({ articles, onClick }) => (
+    <>
+        {articles.map((category, idx) => (
+            <ListWrapper key={idx}>
+                <Header as="h3">{category.category}</Header>
+                <LinkList
+                    link_style={{ size: '2rem' }}
+                    list={category.articles}
+                    onClick={onClick}
+                />
+            </ListWrapper>
+        ))}
+    </>
+)
+ArticleList.propTypes = {
+    articles: PropTypes.array.isRequired,
+    onClick: PropTypes.func,
+}
+
+const ListNoBullets = styled.ul`
+    margin-bottom: 4.2rem;
+    list-style: none;
+
+    > *:not(:last-child) {
+        padding-bottom: 1.6rem;
+    }
+`
+
+const LinkList = ({ list, onClick, link_style }) => (
```

Comment:
reminded me of `LinkedList` C programming 😂

Reply:
`ListWithLinks` better?